var/spack/repos/builtin/packages/flecsi/package.py

```diff
@@ -61,4 +64,10 @@ def build_type(self):
             return 'Release'
 
     def cmake_args(self):
-        return ['-DENABLE_UNIT_TESTS=ON']
+        options = ['-DENABLE_UNIT_TESTS=ON']
```

Comment:
No, it won't run the tests, it just builds them. I just like to build an executable to see if all symbols get resolved.